src/main/java/nextstep/fp/Lambda.java

```diff
@@ -18,39 +18,20 @@ public static void printAllLambda(List<Integer> numbers) {
     }
 
     public static void runThread() {
-        new Thread(new Runnable() {
-            @Override
-            public void run() {
-                System.out.println("Hello from thread");
-            }
+        new Thread(() -> {
+            System.out.println("Hello from thread");
         }).start();
```

Comment:
```suggestion
        new Thread(() -> System.out.println("Hello from thread"))
                .start();
```
한 줄인 경우 중괄호를 생략하기도 합니다